resource-timing/resource_TAO_multi.htm

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8" />
+<title>Resource Timing TAO tests</title>
+<link rel="author" title="Google" href="http://www.google.com/" />
+<link rel="help" href="https://www.w3.org/TR/resource-timing-2/#timing-allow-origin"/>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/webperftestharness.js"></script>
+<script src="resources/webperftestharnessextension.js"></script>
+<script>
+setup({explicit_done: true});
+
+// explicitly test the namespace before we start testing
+test_namespace("getEntriesByType");
+
+var d;
+var iframe;
+var iframeBody;
+var image;
+var random = Math.random();
+
+function setup_iframe() {
+    iframe = document.getElementById('frameContext');
+    d = iframe.contentWindow.document;
+    iframeBody = d.body;
+    iframe.addEventListener('load', onload_test, false);
+}
+function onload_test() {
+    if (window.performance.getEntriesByType === undefined) {
+      done();
+      return;
+    }
+    var context = new PerformanceContext(iframe.contentWindow.performance);
+    var entries = context.getEntriesByType('resource');
+
+    if(entries.length > 0) {
+        entry = entries[0];
+
+        test_equals((entry.redirectStart + entry.redirectEnd + entry.domainLookupStart + entry.domainLookupEnd + entry.connectStart + entry.connectEnd + entry.secureConnectionStart + entry.requestStart + entry.responseStart), 0, 'redirectStart, redirectEnd, domainLookupStart, domainLookupEnd, connectStart, connectEnd, secureConnectionStart, requestStart, and responseStart -- should be all returned as 0 when the HTTP response includes more than one Timing-Allow-Origin header value and TAO algorithm fails, if not overwrite any existing TAO values when appending the new value');
+    }
+
+    done();
+}
+window.setup_iframe = setup_iframe;
+</script>
+</head>
+<body>
+<h1>Description</h1>
+<p>This test validates that for a cross origin resource, the timing allow check algorithm will fail when the HTTP response includes more than one Timing-Allow-Origin header value (not overwrite any existing TAO values when appending the new value).</p>
```

Comment:
Oops, by multi here I meant more than one "case-sensitive match or wildcard"... now that I look at L1 and L2 again, my understanding is:

- In L1, no matter those TAO values match or not, the algorithm will fail; -- if that's the case, we should either update the TAO section in L1 or remove it, because it's broken and it doesn't match the implementation;
- In L2 and L3, this test case should pass, as all the value in the list are either a match or `*`; but there can be two more cases for the value list:
1) if all the values in the list don't match and all the values aren't `*`, it should fail;
2) if any of values in the list is `*` or is a match, then even the other values doesn't match, it should still pass.

@igrigorik does it make sense?